introduction.md

```diff
@@ -1 +1,19 @@
-I’m an architect and web developer, passionate about technology and innovation, with a proven track record as a Project Manager of large, complex projects, who has 27 plus years of professional practice, which include 15 plus years of IT infrastructure experience in a broad range of IT areas such as IT infrastructure, help desk operations, hardware/software upgrades, and 5 plus years of web coding and programming development experience. I have completed a Master degree of Business Administration at the University of Phoenix, a Master degree of Architecture and a Bachelor of Environmental Design degree at the University of Puerto Rico. I have strong organizational and time management skills to juggle multiple tasks within the constraints of timelines.  Proven ability to support the company’s business objectives and to successfully manage all phases of a project.  Strong work responsibility and excellent organizational skills. Fluent in English and Spanish. My favorite Code School path is Javascript and Python.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?